package.json

```diff
@@ -36,7 +36,7 @@
     "cfpb-chart-builder": "3.2.1",
     "del": "3.0.0",
     "glob-all": "3.1.0",
-    "gulp": "3.9.1",
+    "gulp": "4.0.0",
```

Comment:
Any idea why `"gulp": "4.0.0"` would actually install Gulp 4.0.0 when 4.0.0 hasn't been published to [npm](https://www.npmjs.com/package/gulp) yet? 3.9.1 is the latest published as of the time of this comment.

Reply:
Well that's a mystery. I have no idea.

Reply:
We figured this out while you were gone. I forget the answer. Something to do with 4.0.0 being a prerelease or something?

Reply:
It was published as `next`, which @Scotchester I wonder if that's what we should do with the icons branch??

Reply:
That's a very interesting thought, @jimmynotjim...